src/confluent_kafka/admin/__init__.py

```diff
@@ -1204,3 +1234,71 @@ def list_offsets(self, topic_partition_offsets, **kwargs):
 
         super(AdminClient, self).list_offsets(topic_partition_offsets_list, f, **kwargs)
         return futmap
+
+    def delete_records(self, topic_partition_offsets_list, **kwargs):
+        """
+        Deletes all the records before the specified offset,
+        in the specified Topic and Partition.
+
+        :param list(TopicPartitions) topic_partitions_offset_list: A list of TopicPartition objects
+        consisting of the Topic Partition and Offsets on which we have to perform the deleteion.
+        :param float request_timeout: The overall request timeout in seconds,
+                    including broker lookup, request transmission, operation time
+                    on broker, and response. Default: `socket.timeout.ms*1000.0`
+        :param float operation_timeout: The operation timeout in seconds,
+                  controlling how long the DeleteRecords request will block
+                  on the broker waiting for the partition creation to propagate
+                  in the cluster. A value of 0 returns immediately. Default: 0
+
+        :returns: A dict of futures keyed by the TopicPartition.
+                    The future result() method returns DeleteRecordsResult
+                    or raises KafkaException
+
+        :rtype: dict[TopicPartition, future]
+
+        :raises KafkaException: Operation failed locally or on broker.
+        :raises TypeError: Invalid input type.
+        :raises ValueError: Invalid input value.
+        """
+        AdminClient._check_delete_records(topic_partition_offsets_list)
+
+        f, futmap = AdminClient._make_futures_v2(
+            topic_partition_offsets_list, _TopicPartition, AdminClient._make_futmap_result)
+
+        super(AdminClient, self).delete_records(topic_partition_offsets_list, f, **kwargs)
+        return futmap
+
+    def elect_leaders(self, election_type, topic_partition_list, **kwargs):
+        """
+        Perform Preferred or Unclean elections for,
+        all the specified topic partitions.
+
+        :param election_type: Preferred(0) or Unclean(1) elections
```

Comment:
Create Enum for election_type. Don't directly use 0 or 1. See any other Enum for implementation and use.